src/core/sys/posix/arpa/inet.d

```diff
@@ -124,6 +124,31 @@ else version( FreeBSD )
     const(char)*    inet_ntop(int, in void*, char*, socklen_t);
     int             inet_pton(int, in char*, void*);
 }
+else version( NetBSD )
```

Comment:
I am aware that druntime is inconsistent everywhere about this.  But I'd rather all new code follow the same (standard?) convention for version statements.

```
else version (NetBSD)
{
```